README.md

```diff
@@ -1 +1,235 @@
+
+<pre>
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "attributeId",
+                        "values": [
+                            {
+                                "_id": {
+                                    "origin": "2015-02-18T02:46:00.000Z",
+                                    "range": "minute",
+                                    "resolution": "second"
+                                },
+                                "points": [
+                                    {
+                                        "offset": 13,
+                                        "samples": 1,
+                                        "sum": 34.59
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "id": "entityId",
+                "isPattern": false
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+</pre>
+
+In this example response, aggregated time series information for a range of minutes and a resolution of seconds is returned.
+This information has as its origin the 46nd minute, of the 2nd hour of February, the 18th, 2015. And includes data for the
+13th second, for which there is a sample and the sum (and value of that sample) is 34.59.
+
+[Top](#section0)
+
+##<a id="section2"></a> Dependencies
+The STH component is a Node.js application which depends on certain Node.js modules as stated in the ```project.json``` file.
+
+Apart from these Node.js modules, the STH component also needs a running MongoDB instance where the aggregated time series
+information is stored for its proper functioning. Since the STH component uses MongoDB update operators (see
+<a href="http://docs.mongodb.org/v2.6/reference/operator/update/" target="_blank">http://docs.mongodb.org/v2.6/reference/operator/update/</a>)
+such as the ```$max``` and the ```$min``` update operators which were introduced in version 2.6, there is a dependency
+of the STH component with this concrete version of the MongoDB instance where the aggregated data will be stored.
+Consequently, a MongoDB instance version &gt;= 2.6 is needed to store the aggregated time series information.
+
+[Top](#section0)
+
+##<a id="section3"></a> Installation
+1. Clone the repository:
+<pre> git clone https://github.com/telefonicaid/IoT-STH.git </pre>
+2. Get into the directory where the STH repository has been cloned:
+<pre> cd IoT-STH/ </pre>
+3. Install the Node.js modules and dependencies:
+<pre> npm install </pre>
+The STH component server is ready to be started.
+
+[Top](#section0)
+
+##<a id="section4"></a> Inserting data (single events and aggregated data) into the database
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/422dde2e4df20d47b972c2dc549664b1254c9cd5